### PR TITLE
Bugfix/720 search number fix verigreen

### PR DESCRIPTION
**Pull request by AndreeaDora**

### CODE REVIEW

**Comment by lirantal** [comment]
@AndreeaDora would you please care to squash your commits to one for a cleaner git history?

File: verigreen-collector-webapp/src/main/webapp/controllers.js [hunk 1]
```diff
@@ -306,15 +309,18 @@ app.controller('ctrlRead', ['$scope','$filter','$http', 'ModalService', 'sharedP
 
 		$scope.filteredItems = $filter('filter')($scope.items, function(item) {
 			for (var attr in item) {
```
Comment (rqc) [inline verigreen-collector-webapp/src/main/webapp/controllers.js:311]: @AndreeaDora you might want to try changing this to a more reliable loop construct for object _attributes_ as we talked about a couple of weeks back. Please take a look at this: https://docs.angularjs.org/api/ng/function/angular.forEach

**Comment by rqc** [comment]
@AndreeaDora I agree with @lirantal, let's do that for future work. Let me know if you need some pointers on this. Also, I would recommend working on separate feature branches for features that are **not** dependent on each other (do not build on each other). For instance, this fix is for the UI, but the previous commits were related to the email messages. 

Avoiding this would make pull requests smaller and focused and easier to review.

Thanks!

**Comment by AndreeaDora** [comment]
Pull request closed. Another pull request submitted with the search fix and forEach changes.
https://github.com/Verigreen/verigreen/pull/15

**Pull request closed without merging**